packages/webview_flutter/lib/src/webview_android.dart

```diff
@@ -47,10 +47,6 @@ class AndroidWebView implements WebViewPlatform {
               id, webViewPlatformCallbacksHandler));
         },
         gestureRecognizers: gestureRecognizers,
-        // WebView content is not affected by the Android view's layout direction,
-        // we explicitly set it here so that the widget doesn't require an ambient
-        // directionality.
-        layoutDirection: TextDirection.rtl,
```

Comment:
have you tested it with older Android versions? https://github.com/flutter/plugins/pull/1618/files suggests that this is necessary for some version of Android.

Reply:
I'm not seeing anywhere in that PR that specifically talks about this choice, or any indication that it would affect older Android versions. I'll do my best to test on a couple versions of Android though.

Reply:
I've tested on M, O, and R, and haven't seen any negative side effects.

Reply:
Unsurprisingly, that introduces the inverse problem that in an RTL locale, the scroll bars are on the wrong side.

Reply:
We could also use https://api.flutter.dev/flutter/widgets/Directionality/maybeOf.html to detect the current direction, and if `null` we set LTR as the default. wdyt?

Reply:
Oooh that's a neat trick. I left the default as RTL to minimize side effects, but I can swap to LTR if you'd prefer.